stories/components/forms/password-input.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryFn } from "@storybook/react"
+import { PasswordInput } from "@yamada-ui/react"
+import { useState } from "react"
+
+type Story = StoryFn<typeof PasswordInput>
+
+const meta: Meta<typeof PasswordInput> = {
+  component: PasswordInput,
+  title: "Components / Forms / PasswordInput",
+}
+
+export default meta
+
+export const basic: Story = () => {
+  return <PasswordInput />
+}
+
+export const isDisabled: Story = () => {
+  return <PasswordInput isDisabled />
+}
+
+export const withSize: Story = () => {
+  return (
+    <>
+      <PasswordInput size="xs" />
+      <PasswordInput size="sm" />
+      <PasswordInput size="md" />
+      <PasswordInput size="lg" />
+      <PasswordInput size="xl" />
+    </>
```

Comment:
I don't know why the size does not change if I pass props like this.
Can you explain why it not work ,please @hirotomoyamada

Reply:
You have to set `size` in `packages/theme/src/components/password-input.ts`.

Reply:
Oh , just notice that I'am missing an _ in the _ css